main.go

```diff
@@ -17,7 +17,8 @@ import (
 )
 
 const (
-	pathEnv = "PARAMETER_STORE_EXEC_PATH"
+	pathEnv  = "PARAMETER_STORE_EXEC_PATH"
+	forceEnv = "PARAMETER_STORE_FORCE_ENV"
```

Comment:
Prefix should be `PARAMETER_STORE_EXEC_` rather than just `PARAMETER_STORE_`.
Also, how about “overwrite” rather than “force”, as it's a bit more specific / descriptive?

So the env would be `PARAMETER_STORE_EXEC_OVERWRITE`.

```suggestion
	overwriteEnv = "PARAMETER_STORE_EXEC_OVERWRITE"
```

Reply:
Thanks! I've pushed the change.